webxdc.d.ts

```diff
@@ -47,7 +57,10 @@ interface Webxdc<T> {
    * Note that own status updates, that you send with {@link sendUpdate}, also trigger this method
    * @returns promise that resolves when the listener has processed all the update messages known at the time when `setUpdateListener` was called.
    * */
-  setUpdateListener(cb: (statusUpdate: ReceivedStatusUpdate<T>) => void, serial?: number): Promise<void>;
+  setUpdateListener(
+    cb: (statusUpdate: ReceivedStatusUpdate<T>) => void,
+    serial?: number
+  ): Promise<void>;
```

Comment:
This is just formatting, right?

Reply:
yes, would make sense if we made it a habit to format this file with prettier